kits/AGENT/agent-failure-investigator/css/style.css

```
@@ -0,0 +1,442 @@
+/* Agent Failure Investigator — diagnostic console theme.
```

Stylelint (17.14.0)

[error] Unknown rule scss/at-rule-no-unknown. Did you mean at-rule-no-unknown?

(scss/at-rule-no-unknown)